runner/src/hasura-client/hasura-client.ts

```diff
@@ -123,12 +123,81 @@ export default class HasuraClient {
   }
 
   async createSchema (source: string, schemaName: string): Promise<any> {
-    return await this.executeSql(`CREATE schema ${schemaName}`, {
+    //toFix: change back and remove Drop Schema
+    return await this.executeSql(`DROP SCHEMA IF EXISTS ${schemaName} CASCADE;CREATE schema ${schemaName}`, {
       source,
       readOnly: false,
     });
   }
 
+  async createLogs (source: string, schemaName: string): Promise<any> {
```

Comment:
```suggestion
  async createLogsTable (source: string, schemaName: string): Promise<any> {
```
This seems more clear